app/renuo_upload_signing.rb

```diff
@@ -36,6 +32,14 @@ class RenuoUploadSigning < Sinatra::Base
 
   private
 
+  def private_key_valid
+    @private_key_valid ||= settings.authentication.private_api_key_valid?(api_key, params[:private_api_key])
+  end
+
+  def api_key
```

Comment:
i would rename it to `api_key_valid?`

Reply:
Disagree. This method returns the API key, not a boolean result.

Reply:
ou....yes you are right 👍